boards/arm/nrf52840_mdk_m2_module/Kconfig.defconfig

```diff
@@ -0,0 +1,42 @@
+# nRF52840 MDK USB Dongle board configuration
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# Copyright (c) 2022 Nikola Trifunovic
+#
+# Copyright (c) 2022 Matt Anderson
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52840_MDK_M2_MODULE
+
+config BOARD
+	default "nrf52840_mdk_m2_module"
+
+# To let the nRF5 bootloader load an application, the application
+# must be linked after Nordic MBR, that is factory-programmed on the board.
+
+# Nordic nRF5 booatloader exists outside of the partitions specified in the
+# DTS file, so we manually override FLASH_LOAD_OFFEST to link the application
+# correctly, after Nordic MBR.
+
+# When building MCUBoot, MCUBoot itself will select USE_DT_CODE_PARTITION
+# which will make it link into the correct partition specified in DTS file,
+# so no override is necessary.
+
+config FLASH_LOAD_OFFSET
+	default 0x1000
+	depends on BOARD_HAS_NRF5_BOOTLOADER && !USE_DT_CODE_PARTITION
+
+if USB_DEVICE_STACK
+
+# Enable UART driver, needed for CDC ACM
+config SERIAL
+	default y
+
+endif # USB_DEVICE_STACK
```

Comment:
```suggestion
# Enable UART driver, needed for CDC ACM
config SERIAL
	default y if USB_DEVICE_STACK
```

Reply:
fixed in [d0f9bd7](https://github.com/zephyrproject-rtos/zephyr/pull/43992/commits/d0f9bd7f2a6b1c6a706b80ed9617d1cd5b31e48a)